recipes/pyindiclient/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyindi-client" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 76a9d1830fdf9d51b53ec9a8f3e8caed68f8efd0fa44c9071eca753bd9ab0163
+  patches:
+    - libindiclient_path.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - swig
```

Comment:
swig is probably a host dep, like cython

Reply:
changed and pushed